src/include/apparmor/config_complain.rb

```diff
@@ -139,6 +132,21 @@ def getProfModeForm(recList, showAll)
         allText = _("Configure Mode for All Profiles")
       end
 
+      translation_mapping = {
+        # translators: string is value in table for mode of apparmor
+        "enforce"  => _("enforce"),
+        "complain" => _("complain"),
+      }
+
+      recListTranslated = []
+      Builtins.foreach(@recList) do |record|
+        recListTranslated = Builtins.add(
+          recListTranslated,
+          Item(record.params[0], record.params[1],
+            translation_mapping[record.params[2]])
+        )
+      end
```

Comment:
you can use more ruby approach instead of this ycp one like
```
recListTranslated = (@recList || []).map do |record|
  Item(record.params[0], record.params[1], translation_mapping[record.params[2]] || record.params[2])
end
```